Added documentation describing manual signing requirement for Android packages (#1813)
beeware/briefcase: docs/reference/platforms/android/gradle.rst

can be submitted as pull requests. Or, if you have a particular package that you'd like
us to support, please visit the `issue tracker
<https://github.com/chaquo/chaquopy/issues>`__ and provide details about that package.

While it is possible to use `briefcase package android` to produce an APK or AAB
file for distribution, the file is *not* usable as-is. It must be signed
regardless of whether you're distributing your app through the Play Store, or

> ad-hoc via sideloading an APK. For details on how to manually sign your code,

Reviewer: ```suggestion
via loading the APK directly. For details on how to manually sign your code,
```

can be submitted as pull requests. Or, if you have a particular package that you'd like
us to support, please visit the `issue tracker
<https://github.com/chaquo/chaquopy/issues>`__ and provide details about that package.

> Additionally: while it is possible to use `briefcase package android` to produce an APK or
> AAB file for distribution, the file is NOT usable as-is. It must be manually signed regardless
> of whether you're distributing your app by means of the Play Store or ad-hoc via APK. For details on
> how to manually sign your code, see `Sign the Android App Bundle <https://briefcase.readthedocs.
> io/en/stable/how-to/publishing/android.html#sign-the-android-app-bundle>`__.

Reviewer: ```suggestion
While it is possible to use `briefcase package android` to produce an APK or AAB
file for distribution, the file is *not* usable as-is. It must be signed
regardless of whether you're distributing your app through the Play Store, or
ad-hoc via sideloading an APK. For details on how to manually sign your code,
see the instructions on `signing an Android App Bundle
<https://briefcase.readthedocs.io/en/stable/how-to/publishing/android.html#sign-the-android-app-bundle>`__.```
